apps/transport/lib/jobs/resource_history_job.ex

```diff
@@ -229,13 +264,31 @@ defmodule Transport.Jobs.ResourceHistoryJob do
     Map.get(map, key) || Map.get(map, to_string(key))
   end
 
-  defp store_resource_history!(%DB.Resource{datagouv_id: datagouv_id, id: resource_id}, payload) do
-    Logger.debug("Saving ResourceHistory for resource##{resource_id}")
+  defp store_resource_history!(%DB.Resource{} = resource, payload) do
+    %DB.ResourceHistory{
+      datagouv_id: resource.datagouv_id,
+      resource_id: resource.id,
+      payload:
+        Map.merge(payload, %{
+          resource_url: resource.url,
+          resource_latest_url: resource.latest_url,
+          title: resource.title,
+          format: resource.format,
+          dataset_id: resource.dataset_id,
+          schema_name: resource.schema_name,
+          schema_version: resource.schema_version,
+          latest_schema_version_to_date: latest_schema_version_to_date(resource)
+        }),
+      last_up_to_date_at: DateTime.utc_now()
+    }
+    |> DB.Repo.insert!()
+  end
 
+  defp store_resource_history!(%DB.ReuserImprovedData{id: reuser_improved_data_id}, payload) do
     %DB.ResourceHistory{
-      datagouv_id: datagouv_id,
-      resource_id: resource_id,
-      payload: payload,
+      datagouv_id: "reuser_improved_data::#{reuser_improved_data_id}",
```

Comment:
C'est `not null` donc je mets cette valeur, qui est discutable

Reply:
Peut-être faudrait-il renommer la colonne `datagouv_id` en quelque chose de plus général (`source_id`, `external_id`?) et namespacer systématiquement avec `datagouv:` pour les valeurs traditionnelles. Je ne suggère pas de faire ça ici.